ios-gigs/ios-gigs/View Controllers/GigsTableViewController.swift

```diff
@@ -0,0 +1,86 @@
+//
+//  GigsTableViewController.swift
+//  ios-gigs
+//
+//  Created by Ryan Murphy on 5/16/19.
+//  Copyright © 2019 Ryan Murphy. All rights reserved.
+//
+
+import UIKit
+
+class GigsTableViewController: UITableViewController {
+var gigController = GigController()
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+    }
+
+    override func viewDidAppear(_ animated: Bool) {
+        super.viewDidAppear(true)
+        if gigController.bearer == nil {
+            performSegue(withIdentifier: "LoginScreen", sender: self)
+        } else { gigController.fetchGigs { (result) in
+            DispatchQueue.main.async {
```

Comment:
You needed to get the (result) which will have all your Gigs and set the gigController.gigs to the result of gigs.

```
    override func viewDidAppear(_ animated: Bool) {
        super.viewDidAppear(true)
        if gigController.bearer == nil {
            performSegue(withIdentifier: "LoginScreen", sender: self)
        } else { gigController.fetchGigs { (result) in
            
            do {
                self.gigController.gigs = try result.get()
                DispatchQueue.main.async {
                    self.tableView.reloadData()
                }
            } catch {
                NSLog("Error fetching all gigs: \(error.localizedDescription)")
            }
            
    }
        }
    }
```